client/src/containers/Workspace/Workspace.js

```diff
@@ -0,0 +1,352 @@
+<<<<<<< HEAD
```

Comment:
merge conflicts? You using husky?Have you tried running the code aftern mergings?

Reply:
Sorry, this just cropped up, will fix.

Reply:
@nimish-gupta , fixed.